Cargo.toml

```diff
@@ -1,10 +1,12 @@
 [package]
-name = "line-runner"
+name = "line_runner"
```

Comment:
I don't know what's conventional here - I thought I saw Rust example projects being created with kebab-case names but then writing a `use` statement with kebab-case seems impossible?

Reply:
People are all over the place with it. On https://crates.io/ I see a mix with kabob maybe preferred?

Reply:
Ah ok per eg [this comment](https://github.com/rust-lang/api-guidelines/discussions/29#discussioncomment-233422), it knows how to resolve eg `use line_runner::...` in `src/main.rs` to `src/lib.rs` based on the kebab-case `line-runner` package name here

So restored to kebab-case